packages/Webkul/BookingProduct/src/Type/Booking.php

```diff
@@ -213,6 +213,7 @@ public function prepareForCart($data)
                 return trans('shop::app.checkout.cart.integrity.missing_options');
             }
 
+            $tmp = [];
```

Comment:
Give some meaningful names here and a new line.

Reply:
OK